core/cmd/shell_local.go

```diff
@@ -604,7 +612,7 @@ func (s *Shell) RebroadcastTransactions(c *cli.Context) (err error) {
 		return s.errorOut(errors.Wrap(err, "fatal error instantiating application"))
 	}
 
-	chain, err := app.GetChains().EVM.Get(chainID)
+	chain, err := app.GetRelayers().LegacyEVMChains().Get(chainID.String())
```

Comment:
Do we have a story tracking these kinds of exceptions that need to be pushed inside the relayer to be supported for LOOPs? `SendTx` and chain/node statuses were the only relevant ones for solana/starknet, and didn't have as main chain-specific options as this one.

Reply:
no we don't have a story, i can make one.

i'm wondering if bad naming is at play here. GetRelayers is returning a [RelayerChainInteroperators], which is glue between a relayer and chain in the case of EVM while we don't have an EVM Loopp implementation. 

I imagine LegacyEVMChains going away entirely when the products are chain agnostic. That is, it's not clear to me that this is an exception that need to be accommodated in the relayer interface, rather it seems like a hold over that is needed now because products are directly manipulating the EVM chain.

Alternatively it's possible i'm misunderstanding your comment here. Are you concerned about the form of the API as I've written it or are you specifically concern how RebroadcastTxn is implemented the implications that has for the generic relayer interface?

Reply:
I just meant that if we need to support this long-term for LOOP mode, then we'll have to add it to the Relayer API. I think this is fine for now. I do suspect we'll need some version of it though.